_sass/_layout.scss

```diff
@@ -751,28 +790,8 @@ div.facebook {
         display: none;
     }
 }
-
 // Bootstrap Overwrites
-table.sortable > thead th:hover:not([data-defaultsort="disabled"]) {
-  background-color: $brand-color-light;
-}
-
-@include media-query($on-laptop) {
```

Comment:
this here should be in the PR as well (see comment above). Otherwise the menu looks really weird on md devices